celesta-sql/src/main/java/ru/curs/celesta/dbutils/adaptors/PostgresAdaptor.java

```diff
@@ -167,11 +175,12 @@ public DbColumnInfo getColumnInfo(Connection conn, Column c) {
                             String tableName = c.getParentTable().getName();
 
                             if (sequenceName.equals(tableName + "_seq")) {
+                                // PP: probably dead code 
```

Comment:
just remove the whole `if (sequenceName.equals(tableName + "_seq"))` thing

now we don't have any 'magic' sequence names and should treat all the sequences equally